plugins/put-a-file-with-ftp/impl/src/main/java/org/pentaho/di/job/entries/ftpput/JobEntryFTPPUT.java

```diff
@@ -642,13 +642,18 @@ public Result execute( Result previousResult, int nr ) {
         }
       }
 
-      Path path = Paths.get( realLocalDirectory );
-      if ( !Files.isReadable( path ) || !Files.isWritable( path ) ) {
-        throw new FTPException( BaseMessages.getString( PKG, "JobFTPPUT.LocalDir.NoPermission" ) );
+      boolean isStandardFileSystem = (!realLocalDirectory.contains("://")) || realLocalDirectory.equalsIgnoreCase("file");
```

Comment:
is the closing bracket at `"://"))` supposed to end of line ?
and the spaces should be enclosed with brackets `( !realLocalDirectory.contains( "://" )`